lib/chromic_pdf/pdf/template.ex

```diff
@@ -0,0 +1,172 @@
+defmodule ChromicPDF.Template do
+  @moduledoc """
+  This module contains basic styles and options that make it simple to make the HTML content fully
+  cover a given page.
+  """
```

Comment:
This does not sound fully clear to me.
I would probably mention that this module allows you to create a reusable template with custom content such header, footer, style and even content.

Something like:

"Use this module to create your HTML template with a custom header, footer, content and style.

or

"This module exposes helper methods to create your own HTML page template, including header, footer, content and style."


Feel free to ignore my comment if it is incorrect 😉

Reply:
:) hmm, "create your own HTML page template" -> I would guess there is more to it than this file, so I'd like to keep it closer to "these are some helpers that help configure the page layout and margins" or so. Nevertheless, it's unclear, let me rephrase it 👍